python/ccf/proposal_generator.py

```diff
@@ -18,6 +18,9 @@
 from loguru import logger as LOG  # type: ignore
 
 
+GENERATE_JS_PROPOSALS=os.getenv("JS_GOVERNANCE")
```

Comment:
Maybe add a CLI flag instead of the env var?

Reply:
I prefer the env var since its simpler to turn on globally for an entire test (ie - at some point soon we can switch export this env var for our e2e suites, and they should all work with the new proposal style). The CLI invocations of this are pretty rare, so I don't think its worse to have the env var than a flag during the transition period.